Agreements/Internal/TCCOW_TCNW.txt

```diff
@@ -14,3 +14,8 @@ COPX:*:*:CLIPY:EGMC:*:London TC COWLY:London TC NW:*:15000:CLIPY
 COPX:*:*:CLIPY:EGSS:*:London TC COWLY:London TC NW:*:15000:CLIPY
 COPX:*:*:CLIPY:EGSC:*:London TC COWLY:London TC NW:*:15000:CLIPY
 COPX:*:*:CLIPY:EGGW:*:London TC COWLY:London TC NW:*:15000:CLIPY
```

Comment:
Would we be able to move the CLIPY agreements into an 'Archived 2021/05' section at the bottom - just so it's obvious why we left them in

Reply:
done